lib/schedule.rb

```diff
@@ -0,0 +1,10 @@
+def schedule(time='1h', first_in=10)
+  
+  SCHEDULER.every time, :first_in => first_in+rand(30) do
```

Comment:
the random factor is there to jitter the start times so that not everything hits at once, and so that the order is a bit random each time.